app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+  before_action :set_boat, only: [ :new, :create ]
+
+  def new
+    @review = Review.new
```

Comment:
Shouldn't we make a reference to the boat that is reviewed here? Remember we had an error the other day with the Booking controller at this level.

Reply:
i know but it works this way i am telling you